pynicotine/core.py

```diff
@@ -464,8 +464,8 @@ def _check_privileges(self, msg):
         days = hours // 24
 
         if msg.seconds == 0:
-            log.add(_("You have no Soulseek privileges. Privileges are not required, but allow your downloads "
-                      "to be queued ahead of non-privileged users."))
+            log.add(_("You have no Soulseek privileges. While your privileges are active, your downloads "
```

Comment:
Perhaps `your` could be removed here?

Reply:
Yes, even better. Perhaps add something like 'you can get privileges by donating at: http://www.slsknet.org/donate.php'?

Reply:
The status bar where the message is usually displayed does not support clickable hyperlinks.